shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/java/org/apache/shardingsphere/integration/data/pipeline/framework/container/compose/DockerComposedContainer.java

```diff
@@ -38,15 +38,20 @@ public final class DockerComposedContainer extends BaseComposedContainer {
     
     private final ShardingSphereProxyDockerContainer proxyContainer;
     
+    private final ShardingSphereProxyDockerContainer anotherProxyContainer;
+    
```

Comment:
1, Seems could be local variable in constructor

2, Should `anotherProxyContainer` be closed on stop()?

Reply:
Simplify code